packages/importer/src/importer.ts

```diff
@@ -423,15 +427,32 @@ export class OneSchemaImporterClass extends EventEmitter {
 
         break
       }
+      // Only used for initialization errors
       case "error": {
-        this.emit("error", event.data.message)
+        this.emitErrorEvent({
+          code: OneSchemaErrorCode.INITIALIZATION,
+          message: event.data.message,
+        })
+
         if (this.#params.autoClose) {
           this.close()
         }
         break
       }
+      case "http-error": {
+        this.emitErrorEvent({
+          code: OneSchemaErrorCode.HTTP,
+          message: event.data.data.message,
+        })
+        break
+      }
     }
   }
+
+  emitErrorEvent(payload: OneSchemaError) {
+    console.error(payload.message)
```

Comment:
hmm maybe we only do the console.error in devmode?

Reply:
Eh, it seems fine to log them outside of dev mode. It shouldn't impact the user experience.